protected/humhub/modules/friendship/widgets/FriendshipButton.php

```diff
@@ -20,10 +22,76 @@ class FriendshipButton extends \yii\base\Widget
 {
 
     /**
-     * @var User the target user 
+     * @var User the target user
      */
     public $user;
 
+    /**
+     * @var array Options buttons
+     */
+    public $options = [];
+
+    private function getDefaultOptions()
+    {
+        return [
+            'friends' => [
+                'title' => '<span class="glyphicon glyphicon-ok"></span>&nbsp;&nbsp;' . Yii::t('FriendshipModule.base', 'Friends'),
+                'attrs' => [
+                    'data-method' => 'POST',
+                    'data-confirm' => Yii::t('FriendshipModule.base', 'Would you like to end your friendship with {userName}?', ['{userName}' => '"' . $this->user->getDisplayName() . '"']),
+                    'class' => 'btn btn-info active',
+                ],
+            ],
+            'addFriend' => [
+                'title' => '<span class="glyphicon glyphicon-plus"></span>&nbsp;&nbsp;' . Yii::t('FriendshipModule.base', 'Friends'),
+                'attrs' => [
+                    'data-method' => 'POST',
+                    'data-confirm' => Yii::t('FriendshipModule.base', 'Would you like to send a friendship request to {userName}?', ['{userName}' => '"' . $this->user->getDisplayName() . '"']),
```

Comment:
Html::encode?

Reply:
answer in https://github.com/humhub/humhub/pull/5057#discussion_r644786497